e2e/chat.spec.ts

```diff
@@ -774,12 +774,10 @@ describe('chat', () => {
 
     await chat.submitMessage('Here are multiple images');
 
-    for (let i = 1; i <= maxFileUploads; i++) {
-      const uploadedImage = page.getByRole('img', {
-        name: `User uploaded image ${i}`,
-      });
-      await expect(uploadedImage).toBeVisible();
-    }
```

Comment:
We won't show multiple image previews at once in the chat anymore, just one as the active one in the carousel